SimpleCalculator.py

```diff
@@ -71,32 +72,43 @@ def find_min_value(values):
 # TODO this function should take list of values, then return value with biggest value
 def find_max_value(values):
     max_value = 0
-    # TODO  here should be your implementation
+
+
     return max_value
 
 
 # TODO this function should take list of values, then return average of those values
 
+
 def calculate_average(values):
     average = 0
     average = sum(values) / len(values)
-    # TODO  here should be your implementation
     return average
 
 
 
 # TODO this function should take list of values,
 # then returns sorted list in ascending(normal) or descending(reverse" order
 def sort_values(values, order="normal"):
-     # TODO  here should be your implementation
+    for x in range(0,5):
```

Comment:
ok look nice, but there is one problem, it only works for list that have 6 elements in it, from index 0 to 5, you should fox it so it will work for list with any length. For e.g you could change to 5 in both range to length of list, you already used len function :) also if you use range(5) you should probably get range(0, 5) so You could delete 0 from range